versioned_docs/version-0.12/user-guide/ingest-data/for-observability/fluent-bit.md

```diff
@@ -0,0 +1,81 @@
+---
+keywords: [Fluent bit, Prometheus Remote Write, OpenTelemetry, data pipeline]
+description: Instructions on integrating GreptimeDB with Fluent bit for Prometheus Remote Write and OpenTelemetry.
+---
+
+# Fluent Bit
+
+[Fluent Bit](http://fluentbit.io/) is a fast and lightweight telemetry agent for logs, metrics, and traces for Linux, macOS, Windows, and BSD family operating systems. Fluent Bit has been made with a strong focus on performance to allow the collection and processing of telemetry data from different sources without complexity.
+
+You can integrate GreptimeDB as a OUTPUT for Fluent Bit.
+
+## Http
+
+Using Fluent Bit's [HTTP Output Plugin](https://docs.fluentbit.io/manual/pipeline/outputs/http), you can send logs to GreptimeDB.
+
+```conf
+[OUTPUT]
+    Name http
+    Match *
+    Host greptimedb
+    Port 4000
+    Uri /v1/events/logs?db=public&table=your_table&pipeline_name=pipeline_if_any
+    Format json
+    Json_date_key scrape_timestamp
+    Json_date_format iso8601
+    Header Authorization "Bearer <token> if any"
+```
+
+- `host`: GreptimeDB host address, e.g., `localhost`.
+- `port`: GreptimeDB port, default is `4000`.
+- `uri`: The endpoint to send logs to.
+- `format`: The format of the logs, needs to be `json`.
+- `json_date_key`: The key in the JSON object that contains the timestamp.
+- `json_date_format`: The format of the timestamp.
+- `header`: The header to send with the request, e.g., `Authorization` for authentication.
```

Comment:
maybe needs to link to `/user-guide/protocols/http.md#authentication`?

Reply:
I've linked the documentation for the entire interface in its entirety below.